pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -45,6 +45,7 @@ message Schema {
         LocalTime = 18;
         LocalDateTime = 19;
         ProtobufNative = 20;
+        AutoConsume = -3;
```

Comment:
I am not sure whether we can define -3 as `AutoConsume`, @codelipenghui  could you please take a look that

Reply:
In SchemaType, AUTO_CONSUME = -3.

Reply:
yes, I know. I just don't know if it is possible to pass negative numbers, I communicated with penghui, you can do this in this way

Reply:
This looks like a wire protocol change (even if it isn't?)

Probably we need further discussion on the mailing list or even a PIP